website/components/_temp-enable-hidden-pages/index.js

```diff
@@ -1,85 +0,0 @@
-// Imports below are used in server-side only
```

Comment:
This was a temporary hot fix for hidden pages, has been replaced by changes in [react-components#220](https://github.com/hashicorp/react-components/pull/220).